7.1.Цикл_for/7.1.7.py

```diff
@@ -1,5 +1,4 @@
-# Повторяй за мной 2
-
-Name = input()
-for i in range(10):
-    print(i, Name)
\ No newline at end of file
+# Повторяй за мной 2
+Name = input()
```

Comment:
Небольшое замечание - по PEP8 переменные должны иметь `snake_case`, то есть начинаться с маленькой буквы и разделяться нижними подчеркиваниями:
```suggestion
name = input()
```